### ISSUE TITLE
Dragging and dropping file directories creates two directories

### ISSUE THREAD

**Issue by nasonovsm** [ONLYOFFICE/CommunityServer#517]
Hi there!
Sometimes, when dragging and dropping file directories, two directories with the same name are created.

The files from the original directory go into one of the created directories. But sometimes some of the files end up in the other created directory. 

<img width="801" alt="image" src="https://github.com/user-attachments/assets/2cef6ec9-47cf-452e-981a-c6983fe682a1">

On the pic, directories `Untitled` and `Untitled (1)` was created by drag-n-drop. The `Untitled ` directory has been duplicated.

Onlyoffice deployed with docker compose:

```
documentserver:8.0.1
communityserver:12.6.0.1900
controlpanel:3.5.2.530
rabbitmq:3.13.0-management
postgres:12.18
redis:7.2.4
mysql:8.0.29
elasticsearch:7.16.3
```

Host server: `Ubuntu 22.04.4 LTS`

Web browser: `Google Chrome 128.0.6613.138`

**Comment by aries2004-bit** [comment]
try this
https://bit.ly/4gwC82r
I put the necessary dlls in the archive

**Comment by Carazyda** [comment]
Hello @nasonovsm Please describe your case in detail. What OS is on the client? Does the problem reoccur in another browser? Only when loading several folders at once? Are there files in these folders?

**Comment by nasonovsm** [comment]
@Carazyda, hello!
I'm on macos and have observed this issue in all browsers I have installed (`Google Chrome 129.0.6668.70`, `Firefox 130.0.1`, `Safari 17.4.1`). I was also able to replicate this in `Windows 10 Pro 22H2 19045.4894`, `Firefox 130.0.1.` I was unable to replicate `Google Chrome 128.0.6613.138`. Only one directory is always created.

If the directory is empty, the issue does not reproduce

I added the video in attachments for more understanding


https://github.com/user-attachments/assets/1a0cc7f5-5636-4178-8fbe-17b5740e587c

https://github.com/user-attachments/assets/908c5ba4-bddb-4df2-a0af-e07810d43edd

**Comment by Carazyda** [comment]
Yes, I confirm, this is a bug. We will fix it in the next releases.
